### ISSUE TITLE
jaiext no longer resolved

### ISSUE THREAD

**Issue by mattwigway** [conveyal/r5#585]
I think that it.geosolutions.jaiext used to be in the OSGeo maven repository, but it isn't any more. I had to add the boundlessgeo repo (linked from [GeoTools docs](https://docs.geotools.org/latest/userguide/tutorial/quickstart/maven.html)), NB it is available over TLS (https) as well as plain HTTP.

**Comment by wlovotti** [comment]
I think I might be having an issue tangentially related to this. When I try to build R5 locally with `mvn clean package` I get the following build error:

`Failed to execute goal on project r5: Could not resolve dependencies for project com.conveyal:r5:jar:4.9.0-SNAPSHOT: Failed to collect dependencies at org.geotools:gt-coverage:jar:21.2 -> it.geosolutions.imageio-ext:imageio-ext-tiff:jar:1.2.2: Failed to read artifact descriptor for it.geosolutions.imageio-ext:imageio-ext-tiff:jar:1.2.2: Could not transfer artifact it.geosolutions.imageio-ext:imageio-ext-tiff:pom:1.2.2 from/to central.maven.org (http://central.maven.org/maven2): Transfer failed for http://central.maven.org/maven2/it/geosolutions/imageio-ext/imageio-ext-tiff/1.2.2/imageio-ext-tiff-1.2.2.pom: Unknown host central.maven.org: nodename nor servname provided, or not known`

The domain `central.maven.org` is no longer active, per https://central.sonatype.org/articles/2019/Jul/15/central-http-deprecation-update/.

**Comment by mattwigway** [comment]
Yes, this is definitely the same issue, I got that same error. It's trying
to fall back on central.maven.org after it doesn't find it in any other
repository.

Matt
